Source/WTF/wtf/SegmentedVector.h

```diff
@@ -222,14 +221,14 @@ namespace WTF {
     private:
         struct Segment {
             T entries[0];
+
+            std::span<T, SegmentSize> entriesSpan() { return unsafeMakeSpan<T, SegmentSize>(entries, SegmentSize); }
         };
 
-        void deleteAllSegments()
+        void destroyAllItems()
         {
             for (size_t i = 0; i < m_size; ++i)
                 at(i).~T();
```

Comment:
Might be worth investigating whether we can use `std::ranges::destroy` here at some point.